release-notes.txt

```diff
@@ -1,3 +1,688 @@
+RIOT-2020.01 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2020.01 release includes:
+
+ - Initial support for micropython
+ - Initial support for GNRC based LoRaWAN stack
+ - Initial asynchronous sock support
+ - Extend support for lpc2387 and fe310
+ - Xtimer concurrency/robustness improvement, fixing #8388, #5338 & #5103
+ - Complete re-implementation of esp8266 based on ESP8266 RTOS SDK
+ - Improvements in automatic tests board coverage and test automation
+ - Introduce and start using kconfig as a configuration tool
+ - +12 new boards, +7 new drivers, +4 new packages
+
+About 526 pull requests, composed of x commits, have been merged since the
+last release, and about 60 issues have been solved. x people contributed with
+code in x days. Approximately x files have been touched with x
+insertions and x deletions.
+
+Notations used below:
+=====================
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
+
+System libraries
+----------------
+
+    + core/kernel_defines: Introduce 'IS_ACTIVE' macro
+    * core/assert: allow multiple static_asserts within a function
+    * core/ringbuffer: compensate 'ringbuffer_remove' underflow error
+    + core/thread: add zombie thread state
+    + sys: Basic C++ compatibility with C11 atomics
+    + sys/arduino: added Wire (I2C) interface
```

Comment:
Would it be possible to somehow group addition/changes of the same area when there are several. Something like:
```
* sys/arduino:
  + added Wire (I2C) interface
  * allow use of arduino libraries...
  etc
```

?